custom/git-app-deployer/main.go

```diff
@@ -0,0 +1,174 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/litmuschaos/test-tools/pkg/log"
+	"github.com/openebs/maya/pkg/util/retry"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+func main() {
+
+	var kubeconfig *string
+	var timeout int
+	file := flag.String("file", "", "absolute path of the sock-shop")
```

Comment:
Do `type` flag will make more sense than file? and you can derive the filePath based on type in beginning and use the same in this file.

Reply:
We can not use type since it is a keyword. Instead of it may I go for typeName?